### PR TITLE
Add a skeleton for a new Recreativos Franco slot game and move all RF drivers to their own folder

### PR DESCRIPTION
**Pull request by clawgrip**
New not working machine
-----------------------------
Jail Break (Recreativos Franco slot machine) [Hammy, ClawGrip]

### CODE REVIEW

File: src/mame/recfranco/rfjailbrk.cpp [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+namespace {
+
+class rfjailbrk_state : public driver_device
+{
+public:
+	rfjailbrk_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_okim6376(*this, "oki")
```
Comment (angelosa) [inline src/mame/recfranco/rfjailbrk.cpp:61]: I dislike this naming. I don't need to remember this being 6376 in code, can as well be short-handed to `m_oki` as also the device tag hints.

Comment (clawgrip) [reply src/mame/recfranco/rfjailbrk.cpp:61]: You mean changing _m_okim6376(*this, "oki")_ to _m_oki(*this, "okim6376")_ ??

Thanks!!!!

Comment (angelosa) [reply src/mame/recfranco/rfjailbrk.cpp:61]: `m_oki(*this, "oki")`

Comment (clawgrip) [reply src/mame/recfranco/rfjailbrk.cpp]: Done, thanks!!!!

**Pull request merged**